.circleci/config.yml

```diff
@@ -63,18 +64,31 @@ workflows:
   version: 2
   build-and-test:
     jobs:
-    - build
-    - test:
-        requires:
-        - build
-        filters:
-          branches:
-            ignore: publish-amis
-    - deploy:
-        requires:
-        - build
-        filters:
-          tags:
-            only: /^v.*/
-          branches:
-            only: publish-amis
+      - build
+      - test:
+          requires:
+            - build
+          filters:
+            branches:
+              ignore: publish-amis
+      - deploy:
+          requires:
+            - build
+          filters:
+            branches:
+              only: publish-amis
+            tags:
+              only: /^v.*/
+  nightly-test:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - publish-amis
```

Comment:
Should this be `ignore` and not `only`? Either way, it'd be helpful to add a comment here on what the nightly test is actually testing: All tests or just the publish-ami functionality.

Reply:
Oh it should be only: - master. Forgot to switch back after I tested.